roofit/roofitcore/src/RooAbsArg.cxx

```diff
@@ -1546,44 +1546,42 @@ void RooAbsArg::printAttribList(ostream& os) const
   if (!first) os << "] " ;
 }
 
+
 ////////////////////////////////////////////////////////////////////////////////
-/// Replace server nodes with names matching the dataset variable names
-/// with those data set variables, making this PDF directly dependent on the dataset.
+/// Replace server nodes with names matching the collection variable names
+/// with those collection variables, making this PDF directly dependent on the dataset.
```

Comment:
`dependent --> depend`?

Reply:
In general, this docstring could be improved. The function name is not so great, so it needs better explanations. How about this?
```
/// Bind this node to objects in `set`.
/// Search the set for objects that have the same name as our servers, and attach ourselves to those.
/// After this operation, this node is computing its values based on the new servers.
/// This can be used to e.g. read values from a dataset.
```